training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+5. Login to concourse as `dp` user (if you do not have the login, please ask anyone in the team)
+
+More information about Concourse can be found on their website - https://concourse-ci.org/docs.html 
+
+## Consul
+Visit the following to access Consul
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://consul.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://consul.production.onsdigital.co.uk
+
+[<img src="images/consul-dev-services.png" title="Consul Develop Home" width=49.7% height=49.7%/>][consul-dev-services] [<img src="images/consul-dev-nodes.png" title="Consul Develop Nodes" width=49.7% height=49.7%/>][consul-dev-nodes]
+
+> Consul is a service mesh solution providing a full featured control plane with service discovery, configuration, and segmentation functionality. Each of these features can be used individually as needed, or they can be used together to build a full service mesh
+
+Consul is mainly used to check the health of applications which Consul retrieves by calling the health check endpoints of the application (`/health`). This information is then used to monitor cluster health, and it is used by the service discovery components to route traffic away from unhealthy hosts
+
+More information about Consul can be found on their website - https://www.consul.io/docs/intro
+
+## Nomad
+Visit the following to access Nomad
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://nomad.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://nomad.production.onsdigital.co.uk
+
+[<img src="images/nomad-dev-home.png" title="Nomad Develop Home" width=49.7% height=49.7%/>][nomad-dev-home] [<img src="images/nomad-dev-job.png" title="Nomad Develop Job Page" width=49.7% height=49.7%/>][nomad-dev-home]
+
+> Nomad is a simple and flexible workload orchestrator to deploy and manage containers and non-containerized applications across on-prem and clouds at scale
+
+Nomad enables us to deploy our applications by bundling computing resources of [AWS][AWS]. In Nomad, each application is considered as a [job][nomad-job] which can have multiple [task groups][nomad-task-group] (`web` or `publishing`). The mapping of these tasks in a job is done using allocations, which is used to declare that a set of tasks in a job should be run on a particular node. Nomad would schedule these allocations which is detailed more [here][nomad-scheduling].   
+
+With Nomad, we can understand whether a particular allocation or the entire service itself is failing. We can look at a specific task depending on the journey (e.g. looking at the publishing task group of the application if the publishing journey was carried out). We can check the logs for specific allocations for debugging. Another common use of Nomad is to stop and start a service (do not do this in production) to capture new updates and secrets for the application. 
```

Comment:
Is it worth mentioning that the logs we can see here are limted, so theres some reason we'd use tools like Kibana later on? :)